silk/model_factory.py

```diff
@@ -66,19 +66,20 @@ def encoded_headers(self):
         """
         From Django docs (https://docs.djangoproject.com/en/2.0/ref/request-response/#httprequest-objects):
         """
-        headers = {}
-        sensitive_headers = {'authorization'}
+        sensitive_headers = SilkyConfig().SILKY_SENSITIVE_KEYS
+
+        sensitive_headers.add('authorization')
+        if SilkyConfig().SILKY_HIDE_COOKIES:
+            sensitive_headers.add('cookie')
+
+        sensitive_headers = map(str.lower, sensitive_headers)
 
+        headers = {}
         for k, v in self.request.headers.items():
             if k in sensitive_headers:
```

Comment:
let's do k.lower() maybe to avoid issues, if smth changes in Django?

Reply:
Oh, really
I missed that